pkgs/development/compilers/gcc/10/default.nix

```diff
@@ -156,7 +156,9 @@ stdenv.mkDerivation ({
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
   nativeBuildInputs = [ texinfo which gettext ]
-    ++ (optional (perl != null) perl);
+    ++ (optional (perl != null) perl)
+    ++ (optional langAda gnatboot)
```

Comment:
```suggestion
    ++ optional (perl != null) perl
    ++ optional langAda gnatboot
```

Reply:
Won't start to reformat the gcc expression in this change, let's keep it consistent in itself.